fastly/config_store_item_test.go

```diff
@@ -0,0 +1,267 @@
+package fastly
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestClient_CreateConfigStoreItem(t *testing.T) {
+	t.Parallel()
+
+	cs := createConfigStore(t)
+
+	var (
+		item *ConfigStoreItem
+		err  error
+	)
+	const value = "testing 123"
+
+	record(t, fmt.Sprintf("config_store_item/%s/create_item", t.Name()), func(c *Client) {
+		item, err = c.CreateConfigStoreItem(&CreateConfigStoreItemInput{
+			StoreID: cs.ID,
+			Key:     t.Name(),
+			Value:   value,
+		})
+	})
+	if err != nil {
+		t.Fatalf("error creating config store item: %v", err)
+	}
+
+	if got, want := item.StoreID, cs.ID; got != want {
```

Comment:
Just for my own understanding, is it more common to use `have, want` or `got, want` I feel like the former is better due to both words having the same length although `got` is shorter (and that's usually preferred). 

This is more 'bike shedding' than anything so not overly important but from a 'style' perspective I was interested in what we should use for consistency.

Reply:
I don't think this is a particularly strong convention, but `got/want` is mentioned in the [TestComments](https://github.com/golang/go/wiki/TestComments#got-before-want) Go wiki and is what I see most often.

Reply:
Good enough for me 🙂 👍🏻